api4/plugin.go

```diff
@@ -21,7 +21,7 @@ import (
 )
 
 const (
-	MaximumPluginFileSize = 50 * 1024 * 1024
+	MaximumPluginFileSize = 75 * 1024 * 1024
```

Comment:
Hmm, I don't /think/ this is the tunable we want to configure -- IIRC, this doesn't constrain the amount of data `ParseMultipartForm` ingests, just how much it puts into memory vs. storing in temporary files. There's no need to store more in memory for quick access, since the interesting form parts should already be available in the first 50Mb (or something in the request is probably messed up).

When I fixed this locally, I had to bump `FileSettings.MaxFileSize` in the configuration, but I didn't dig deeper to understand the path through which this tunable vs. that one takes effect.

Reply:
Yep, I ran into this and was just about to set the issue as WIP. Will fix.

Reply:
Nevermind, this whole thing is unnecessary -- Claudio bumped up the default to 100MB in August, we're just running old configs. But we should add to the next release's release notes. (for servers that have a similar old config)